app/js/actions/header.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { SET_LOCATIONS, SET_CURRENT_SESSION } from './actionTypes';
+
+const applicationDistribution = window.location.href.split('/')[3];
+const apiBaseUrl = `/${applicationDistribution}/ws/rest/v1`;
+
+/**
+ * @description - fetch openMRS app locations
```

Comment:
Regarding some of these comments, can you take a look at this? https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/

Reply:
Okay, I will look into